### PR TITLE
Nav component

### PR DESCRIPTION
**Pull request by jkhirano**
Created nav bar, each icon/link should load its new page.
Ex - When you click on the Home icon, "Home Page" will appear at the top.
When you click on the Exercise icon, "Exercise Page" will appear at the top.

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ export const LOAD_ACTIVITIES = "LOAD_ACTIVITIES";
 export const LOGIN = "LOGIN";
 export const LOGOUT = "LOGOUT";
 export const REGISTER = "REGISTER";
+export const TOGGLE = "TOGGLE";
```
Comment (danieltichiyama) [inline src/actions/index.js]: I don't see this chain of functions being used anywhere and the component works fine, so I think it can be removed.

File: src/actions/index.js [hunk 2]
```diff
@@ -56,3 +57,10 @@ export const actionsLogout = () => async dispatch => {
       console.log(err);
     });
 };
+
+export const actionsToggle = link => dispatch => {
+  return dispatch({
+    type: TOGGLE,
+    payload: link
+  });
+};
```
Comment (danieltichiyama) [inline src/actions/index.js]: I don't see this chain of functions being used anywhere and the component works fine, so I think it can be removed.

File: src/components/NavigationComponent/NavigationComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { connect } from "react-redux";
+import { actionsToggle } from "../../actions";
```
Comment (danieltichiyama) [inline src/components/NavigationComponent/NavigationComponent.jsx]: I don't see this chain of functions being used anywhere and the component works fine, so I think it can be removed.

File: src/components/NavigationComponent/NavigationComponent.jsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+const mapStateToProps = stores => {
+  return {};
+};
+
+const mapDispatchToProps = dispatch => {
+  return {
+    toggle: link => {
+      return dispatch(actionsToggle(link));
```
Comment (danieltichiyama) [inline src/components/NavigationComponent/NavigationComponent.jsx]: I don't see this chain of functions being used anywhere and the component works fine, so I think it can be removed.

File: src/components/NavigationComponent/NavigationComponent.module.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    font-weight: bolder;
+    img {
+      height: 25px;
+      width: 25px;
+    }
+    &:hover {
+      cursor: pointer;
+    }
+  }
+}
```
Comment (danieltichiyama) [inline src/components/NavigationComponent/NavigationComponent.module.scss]: Something is missing from this.  When the app is shrunk down to mobile size (Galaxy s5 or iPhoneX) the right navigation button is off the screen... I'm trying a bunch of different things but nothing is working.  I'll keep digging.

File: src/reducers/index.js [hunk 6]
```diff
@@ -1,4 +1,4 @@
-import { LOAD_ACTIVITIES, REGISTER, LOGIN, LOGOUT } from "../actions";
+import { LOAD_ACTIVITIES, REGISTER, LOGIN, LOGOUT, TOGGLE } from "../actions";
```
Comment (danieltichiyama) [inline src/reducers/index.js]: TOGGLE - I don't see this chain of functions being used anywhere and the component works fine, so I think it can be removed.

File: src/reducers/index.js [hunk 7]
```diff
@@ -30,6 +30,9 @@ let reducer = (store = initialStore, action) => {
       localStorage.removeItem("session");
       return Object.assign({}, store, { isLoggedIn: false });
 
+    case TOGGLE:
+      return Object.assign({}, store, { display: action.payload });
+
```
Comment (danieltichiyama) [inline src/reducers/index.js]: I don't see this chain of functions being used anywhere and the component works fine, so I think it can be removed.

File: src/routes.js [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+//           path={route.path}
+//           exact={route.exact}
+//           component={route.component}
+//         />
+//       ))}
+//     </>
+//   );
+// };
+
+// export default Routes;
```
Comment (danieltichiyama) [inline src/routes.js]: Since this is all commented out and the component works fine, is this file necessary??

File: src/App/App.js [hunk 9]
```diff
@@ -26,22 +32,33 @@ class App extends Component {
...
-        ) : null}
-      </div>
+          {/* Below is a section for playing with new components, that can be hidden or shown by clicking the button below in the App.  I thought it might help to have it separate until new components are ready to be added to their appropriate place? idk... Daniel */}
+          <button onClick={this.handleHidePlayground}>
+            {this.state.playground ? "Hide Playground" : "Show Playground"}
+          </button>
+          {this.state.playground ? (
+            <div className="component_playground">
+              <SmokeButton></SmokeButton>
+              <NavigationComponent></NavigationComponent>
```
Comment (danieltichiyama) [inline src/App/App.js]: Do you feel like this component is ready to be put into the main app? I feel like it's ready.  If you agree, please replace line 48, ```<AuthorizationPage></AuthorizationPage>``` with your component.

Comment (danieltichiyama) [reply src/App/App.js]: Also, sorry, I just noticed there are merge conflicts with the most recent version of dev, caused mainly by the most recent merge into dev from the news feed component.

File: src/components/NavigationComponent/NavigationComponent.module.scss (hunk 5 above)
Comment (jkhirano) [reply src/components/NavigationComponent/NavigationComponent.module.scss]: Weird..When I chose the Galaxy s5 and iphoneX views, all the icons still show for me..
@rinatoyo @Brianwkhuffman @calmoon808 Would you guys mind taking a look and let me know if this happens on your computers too?

File: src/actions/index.js (hunk 1 above)
Comment (jkhirano) [reply src/actions/index.js]: removed

File: src/actions/index.js (hunk 2 above)
Comment (jkhirano) [reply src/actions/index.js]: removed

File: src/components/NavigationComponent/NavigationComponent.jsx (hunk 3 above)
Comment (jkhirano) [reply src/components/NavigationComponent/NavigationComponent.jsx]: removed both actionsToggle and connect since its not being used

File: src/components/NavigationComponent/NavigationComponent.jsx (hunk 4 above)
Comment (jkhirano) [reply src/components/NavigationComponent/NavigationComponent.jsx]: removed

File: src/reducers/index.js (hunk 6 above)
Comment (jkhirano) [reply src/reducers/index.js]: removed

File: src/reducers/index.js (hunk 7 above)
Comment (jkhirano) [reply src/reducers/index.js]: removed

File: src/routes.js (hunk 8 above)
Comment (jkhirano) [reply src/routes.js]: removed

File: src/App/App.js (hunk 9 above)
Comment (jkhirano) [reply src/App/App.js]: K I replaced it with the nav component. I can handle the merge conflicts while I'm merging right?

**Review by danieltichiyama — COMMENTED** [review]
lgtm, could someone else please take a look to confirm.

File: src/App/App.js (hunk 9 above)
Comment (danieltichiyama) [reply src/App/App.js]: yeah, you're right.

File: src/components/NavigationComponent/NavigationComponent.module.scss (hunk 5 above)
Comment (danieltichiyama) [reply src/components/NavigationComponent/NavigationComponent.module.scss]: yes please check!

**Review by calmoon808 — APPROVED** [review]
everything looked pretty ok to me
